_episodes_rmd/04-r-data-analysis.Rmd

```diff
@@ -753,6 +769,45 @@ The `if_else()` statement reads like, "if country equals "Canada" OR `|` "United
 
 We see that although Canada, the United States, and Mexico account for close to half the population of the Americas, they account for 88% of the CO2 emitted. We just did this math quickly by plugging the numbers from our table into the console to get the percentages. Can we make that a little more reproducible by calculating percentages for population (pop) and total emissions (total) into our data before summarizing?
 
+
+## Finishing with Git and GitHub
+Awesome work! Let's make sure it doesn't go to waste. Time to add, commit, and push our changes to GitHub again - do you remember how?
+
+> ## Exercise: changing directories
+> Print your current working directory and list the items in the directory to check where you are. If you are not in the un-report directory, navigate there. 
+> > Solution: 
+> > ```
+> > pwd  
+> > ls 
+> > cd ~/Desktop/un-report  
+> > ls
+> > ```
+> {: .solution}
+{: .challenge}
+
+> ## Exercise: reviewing git and GitHub
+> 
+> Pull to make sure our local repository is up to date. Then add, commit, and push your commits to GitHub. Don't forget to check your git status periodically to make sure everything is going as expected! 
+> 
+> > Solution: 
+> > ```
+> > git status 
+> > git pull
+> > git status 
+> > git add "data-analysis.R"  
+> > git status 
+> > git commit -m "Create data analysis file"  
+> > git status 
+> > git log --online  
+> > git push
+> > git status 
+> > ```
+> {: .solution}
+{: .challenge}
+
+
+# Bonus exercises
```

Comment:
From looking at these, some of these exercises include only content covered in the lessons and some include new content, right? If that's the case, maybe include a sentence at the beginning of this section explaining that they might have to use the internet to find the answers to some of these?

Also if any are directly related to content above, would be good to move it to that section and call it a bonus exercise (but from a cursory look that doesn't seem to be the case).